ChurchCRM/CRM
# Registration Cleanups

- Registration now POSTS to http://demo.churchcrm.io/register.php  (currently just sends out an email, but will store to DB in future)
- Registration is now done via local API call on the instance of ChurchCRM
- Registration is now AJAX
- API Error Handler is fixed, so that API errors are handled
- Reset the registration flag on already installed DBs
- Added the "Current Church Info" to the registration form
- Include the "email message" that the user enters in the registration POST (this was previously not included)
- Added some gettext()

To test:
* launch the branch in Vagrant, and attempt to register.  @DawoudIO and @crossan007 should get an email

Probably will close #1175.


Closes #1227 #1228
